streams/shared/src/main/scala/zio/stream/ZStream.scala

```diff
@@ -3498,24 +3498,10 @@ final class ZStream[-R, +E, +A] private (val channel: ZChannel[R, Any, Any, Any,
    * Converts this stream into a `scala.collection.Iterator` wrapped in a scoped
    * [[ZIO]]. The returned iterator will only be valid within the scope.
    */
-  def toIterator(implicit trace: Trace): ZIO[R with Scope, Nothing, Iterator[Either[E, A]]] =
```

Comment:
Out of curiosity, why was this change (and the other stream ones) needed?

Reply:
Because they were throwing a `FiberFailure`